docs/paper/admin/how-to/basic-troubleshooting.md

```diff
@@ -0,0 +1,135 @@
+
+If you find any plugin's name in the stack trace, head to [Check Plugin Updates](#check-plugin-updates) and read from there. In most cases, the plugin, whose name is found on the stack trace, is causing the problem. If not, continue reading.
+
+Here are some examples of stack trace.
+
+<details>
+  <summary>Example 01: Server attempted to load chunk saved with newer version of minecraft!</summary>
+
+```plaintext
+[00:00:00 WARN]: java.lang.RuntimeException: Server attempted to load chunk saved with newer version of minecraft! 3218 > 3120
+```
+
+You tried to load the world generated with a higher version of Minecraft. You cannot do this.  
+If you don't have any backup of your world before the chunk version update, you must use your updated world with a higher version of Minecraft.
+
+</details>
+
+<!-- Another examples -->
+
+## Find Out What Plugin Is Causing The Problem
+
+If you can't find the name of any plugin in the thread dump or stack trace, try these steps.
+
+### Disable All Plugins
+
+To determine if it is a plugin or the paper itself that is causing problems, disable all of your plugins first.
+
+You can disable all of your plugins by renaming the `plugins` directory to something else, such as `plugins-disabled`, or by archiving the `plugins` directory and deleting it.
+
+### Run Server Without Any Plugins
+
+Try to run your server without any plugins. If you rename or delete `plugins` directory, your server will run without any plugins.
+
+If you don't have any issues with this condition, it's the plugin that's causing them.  
+If you still experience problems, head to [Paper Documentation](#paper-documentation) because it might be paper that is creating problems, and most of them are because of a misconfigured paper server.
+
+### Binary Search
+
+To find out what plugin is causing problem, [binary search](https://en.wikipedia.org/wiki/Binary_search_algorithm) is ideal way to do it.
+
+Here is brief animation about binary search.
+
+![Binary Search Example](assets/binary-search.gif)
+
+## Check Plugin Updates
+
+There is a chance that your problem is already fixed in the latest release or latest build of the plugin.
+
+Head to your plugin's official download page and check if you are using the latest build or the latest release of the plugin. If not, update it to the latest version and try to run your server again to see if the problem is fixed or not.
+
+### Update Library Plugin Too
+
+Many plugins use library plugins, and you have to download them and put them in `plugins` directory.
+
+If you don't update them to the latest version or latest build, you might experience problems related to plugins that use the library plugin.
+
+Some library plugins, like ProtocolLib, tell their users to use their latest development build for support of the latest Minecraft version. You should look carefully at the requirements of your plugin.
```

Comment:
If you remove the reference to ProtocolLib from here as i added it above. I did this so that it is immediately clear what a library plugin is at the start, rather than realise here.